src/pageObject.js

```diff
@@ -1,15 +1,34 @@
 'use strict';
 
-class Header {
+const f = 'Conduit logo';
+const y = '[Publish comment] button';
 
+class Header {
+  get logo() {
+    return f;
+  };
 };
 
 class PageObject {
-
+  constructor(url) {
+    this.url = url;
+    this.header = new Header();
+  };
+  clickOnLogo() {
+    return 'Click on the ' + `${f}`;
+  };
 };
 
 class ArticlePage extends PageObject {
-
+  get commentButton() {
+    return y;
```

Comment:
you need to modify url from the parent class using constructor and then use it here